src/explore-education-statistics-frontend/server.js

```diff
@@ -68,6 +68,37 @@ async function startServer() {
 
   const server = express();
 
+  function replaceLastOccurance(input, pattern, replacement) {
+    if (
+      input === undefined ||
+      input === null ||
+      input.length === 0 ||
+      !input.endsWith(pattern)
+    ) {
+      return input;
+    }
+
+    return `${input.slice(0, -pattern.length)}${replacement}`;
+  }
```

Comment:
I very much confess that this abstraction is quite noddy - but technically the "rule of three" applies! :D 

Very happy to revert to repeated lines if anyone feels strongly